index.html

```diff
@@ -10,32 +10,39 @@
     <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <div id="heading">
-      <header class="header-nav">
-        <a href="#" class="logo">MER<span>cy</span></a>
-        <nav id="hamburger">
-          <img src="/assets/images/bars.png" alt="menu" />
-        </nav>
-      </header>
-
-      <section class="heading">
-        <div class="header">
-          <h1>I’m MERCY <br />Glad to see <br />you!</h1>
+    <header class="landing-page">
+      <div class="header-nav">
+        <div class="logo">
+        <a href="#" class="logo-text">MER<span>cy</span></a>
         </div>
+        <nav class="menu">
+          <div class="bars">
+          <img id="hamburger" src="./assets/images/bars.png" alt="menu" />
+          </div>
+          <ul class="nav-items">
+            <li>
+              <a class="nav-link" href="#portfolio">Portfolio</a>
+            </li>
+            <li>
+              <a class="nav-link" href="#about">About</a>
+            </li>
+            <li>
+              <a class="nav-link" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
 
-        <div class="heading-paragraph">
-          <p>
+      <section class="headline">
+          <h1 class="headline-text" id="headline-text">I’m MERCY <br/>Glad to see <br class="break" />you!</h1>
+          <p class="headline-paragraph" id="headline-paragraph">
             I’m a software developer! I can help you build a product , feature
             or website Look through some of my work and experience! If you like
             what you see and have a project you need coded, don’t hestiate to
             contact me.
           </p>
-        </div>
-        <div class="heading-social">
-          <div class="icons-header">
-            <p>LET’S CONNECT</p>
-          </div>
-          <ul class="icons">
+          <p class="heading-social" id="heading-social">LET’S CONNECT</p>
+          <ul class="icons" id="icons">
```

Comment:
- [ ] The github logo is not showing. kindly consider giving the link in `src` like this `"./assets/icons/github.png"`